CHANGELOG.md

```diff
@@ -149,6 +149,7 @@
   [#10288](https://github.com/Kong/kong/pull/10288)
 - Added new span attribute `http.client_ip` to capture the client IP when behind a proxy.
   [#10723](https://github.com/Kong/kong/pull/10723)
+  [#10204](https://github.com/Kong/kong/pull/10204)
```

Comment:
Is this right? I didn't follow the PR, but it does not seem related. If it is, we may want to add a shout-out to the author here as well.